spring/spring4/spring4-common/src/test/java/org/infinispan/spring/session/InfinispanApplicationPublishedBridgeTCK.java

```diff
@@ -64,6 +65,11 @@ public void testEventBridge() throws Exception {
 //      EventsWaiter.assertNumberOfEvents(() -> eventsCollector.getEvents(), SessionDestroyedEvent.class, 2, 10, TimeUnit.SECONDS);
    }
 
+   private void sleepOneSecond() {
+      long oneSecondSleep = System.currentTimeMillis() + 1000;
+      eventually(() -> System.currentTimeMillis() > oneSecondSleep);
```

Comment:
I tried to look into `TimeService` as you suggested but I couldn't find any nice way to hook in.

I could construct a loop using `isTimeExpired` but the end result would be the same. But maybe I missed something...